### DISCUSSION TITLE
"The" collection sorting

### DISCUSSION THREAD

**Discussion by justinxfisher** [meisnate12/Plex-Meta-Manager#355] [Q&A]
I have a template set up with `sort_title` as `zzzzz_<<collection_name>>` so these collections show up at the end of my list.

Is there a way to sort collection names like "The Avengers" as "Avengers, The"  if I'm already using `sort_title` like the above?

**Comment by meisnate12** [comment]
there is not but thats a decent ask

**Reply by justinxfisher** [reply]
Is there another place I should post this question to be added?

**Reply by meisnate12** [reply]
create a feature request for it you can do so with the issues
